mne/io/edf/edf.py

```diff
@@ -324,7 +324,7 @@ def _read_segment_file(data, idx, fi, start, stop, raw_extras, filenames,
     return tal_data
 
 
-def _read_header(fname, exclude):
+def _read_header(fname, exclude,misc=[]):
```

Comment:
You [shouldn't](https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/) use a mutable object as a default argument. Either make it `None`, or an empty tuple.